src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.tsx

```diff
@@ -19,12 +19,14 @@ import {
   Select,
   MadieDiscardDialog,
   Toast,
+  RadioButton,
```

Comment:
I feel this should have been named RadioGroup or something similar. We can refactor some other time.